### ISSUE TITLE
"Value cannot be null. Parameter name: folder" appears when trying to upload to 9.11 Resource Manager via Global Assets

### ISSUE THREAD

**Issue by ajplopez** [dnnsoftware/Dnn.Platform#5362]
## Description of bug
In DNN 9.11, in the new Resource Manager, when trying to upload any file an error appears: "Value cannot be null. Parameter name: folder"

![Screenshot 2022-10-29 at 9 57 35 PM](https://user-images.githubusercontent.com/8751936/198863362-84aeea1d-c265-4948-aa54-347dd21fc5f9.png)

However, if uploaded to the Root of the Global Assets, the upload appears to be successful but the file does not appear in the Resource Manager listing. (I can't check actual FTP of the site)

This only appears to affect Global Assets. (Site Assets works as expected)

## Steps to reproduce
List the precise steps to reproduce the bug:
1. Go to Manage in the PersonaBar
2. Click on Global Assets
3. Choose a folder you want to upload a file to
4. Click "Upload"
5. Drag and drop or select a local file and attempt to upload
6. See error "Value cannot be null. Parameter name: folder"

## Current behavior
When trying to upload any type of whitelisted file (jpg, png, css, etc) to the site via Global Assets, the new Resource Manager throws an error "Value cannot be null. Parameter name: folder"

## Expected behavior
The file should upload and then appear in the resource manager

## Screenshots
![Screenshot 2022-10-29 at 9 57 35 PM](https://user-images.githubusercontent.com/8751936/198863362-84aeea1d-c265-4948-aa54-347dd21fc5f9.png)

## Error information
Admin Logs shows no activity on this error. Nothing is logged.

## Additional context
I'm working on a site where I don't have FTP access for security reasons so I was using Global Assets to upload a CSS file to the theme folder. The site was upgraded to DNN 9.11 and suddenly this method stopped working because I received this error.

I've tried various things like Zipping the file and uploading using "Extract uploaded ZIP files) to no avail.

## Affected version
* [X] 09.11.00 

## Affected browser 

(MacOS)
* [X] Chrome
* [X] Firefox
* [X] Safari
* [X] Microsoft Edge Chromium

**Comment by jeremy-farrance** [comment]
Global Assets - I did a few extra tests here and can confirm. The UI gives the appearance that the files uploaded but are not saved. I even checked on the server and did a search to see if they were stuck in a temp folder somewhere, but no. They do not appear to get saved to the file system at all.
 
I tested again in Site Assets and everything worked as expected.
 
I made one additional discovery that probably isn't a bug, more of a needed feature enhancement... it seemed very unexpected to me that double-clicking a folder did not take you *in* to that folder. Along the same lines, right-clicking a folder, the context menu does not contain an option to Open. If not already on someone's radar screen, should I add this as an Issue, as a Feature Enhancement request?
